IFComp/root/src/ballot/index.tt

```diff
@@ -98,6 +98,66 @@
     </p>
 [% END %]
 
+[% platform = {
+        total => 0
+        adrift => 0
+        inform => 0
+        parchment => 0
+        quixe => 0
+        tads => 0
+        quest => 0
+        alan => 0
+        hugo => 0
+        windows => 0
+        website => 0
+        adventuron => 0
+        choicescript => 0
+        ink => 0
+        texture => 0
+        twine => 0
+        unity => 0
+        other => 0
+    }
+%]
+[% FOR entry IN entries %]
+    [% platform.total = platform.total + 1 %]
```

Comment:
Seems like the total isn't being used for anything?

Reply:
yeah, I was going to put the number of games by platform next to the platform name. Total would show the full count.

e.g. the list would look like:
All (16)
Inform (5)
Twine (3)
ChoiceScript (8)
and so on, but it would take more work to play well with the other two filters.